style/config/_colors.scss

```diff
@@ -0,0 +1,10 @@
+$white: #fff;
```

Comment:
Great! In case you want to have bit more off-white shade, you can just reassign it! 😄